src/Hal/Component/OOP/Extractor/MethodExtractor.php

```diff
@@ -191,7 +191,7 @@ private function extractContent(ReflectedMethod $method, $n, TokenCollection $to
      *
      * @param ReflectedMethod $method
      * @param integer $n
-     * @param TokenCollection $tokens
+     * @param TokenCollection[] $tokens
```

Comment:
The type here is not a "collection of TokenCollection". TokenCollection is an iterator.